src/containers/Programs/index.tsx

```diff
@@ -235,21 +237,42 @@ const Programs = ({
                   </InputAndButtonWrapper>
                 </FiltersSearchWrapper>
               </div>
-              {stateToRender.map((data, index) => {
-                return (
-                  <Thumbnail
-                    key={index}
-                    careerTypes={data.careerType}
-                    index={index}
-                    locations={data.locations}
-                    title={data.programName}
-                    onThumbnailSelection={handleUserSelection}
-                    isSelected={index === selectedProgramIndex}
-                    numberOfReviews={data.reviews.length}
-                    programId={data.id}
-                  />
-                );
-              })}
+              <div>
+                {stateToRender.map((data, index) => {
+                  return (
+                    <Thumbnail
+                      key={data.id}
+                      careerTypes={data.careerType}
+                      index={index}
+                      locations={data.locations}
+                      title={data.programName}
+                      stateToRender={stateToRender}
+                      setStateToRender={setStateToRender}
+                      onThumbnailSelection={handleUserSelection}
+                      isSelected={index === selectedProgramIndex}
+                      numberOfReviews={data.reviews}
+                      programId={data.id}
+                    />
+                  );
+                })}
+                {stateToRender.map((data, index) => {
```

Comment:
Could you please look into this Thumbnail. I think it looks like a double return?

Reply:
yes, deleted